scripts/ipfs/ipfs-setup.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+# This script connects to local mesh peers,
+# and it sets up connection filters based on what networks this node can access.
+# It runs continually, to change IPFS settings as the environment around the node changes.
+
+
+# Wait for IPFS to initalize
+attempts=15
+until [[ $(curl http://localhost:5001/api/v0/id -s 2>/dev/null) || ${attempts} -eq 0 ]]; do
+    sleep 1
```

Comment:
I don't think we need to test so often and I don't think 15 s is enough in some cases?